va/caa.go

```diff
@@ -196,11 +227,36 @@ func (va *ValidationAuthorityImpl) validateCAASet(caaSet *CAASet) (present, vali
 		return true, true
 	}
 
+	// Per RFC 6844 Section 5.3 "issueWild properties MUST be ignored when
+	// processing a request for a domain that is not a wildcard domain". Skip
+	// checking `caaSet.Issuewild` if `wildcard` is false.
+	if wildcard && len(caaSet.Issuewild) > 0 {
+		// We are processing CAA for a wildcard authorization identifier and there
+		// was at least one Issuewild CAA record in the set. This means we must
+		// check that our CAA identity is present in the issueWild set.
+		//
+		for _, caaIssuewild := range caaSet.Issuewild {
+			if extractIssuerDomain(caaIssuewild) == va.issuerDomain {
+				va.stats.Inc("CAA.IssuewildAuthorized", 1)
+				return true, true
+			}
+			// Since our identity is *not* present we must without checking the
+			// `caaSet.Issue` identities because per RFC 6844 Section 5.3 if the domain
+			// is a wildcard domain and there is at least one issuewild property then
+			// "all issue properties MUST be ignored"
+			return true, false
+		}
+	}
```

Comment:
There needs to be a return here, if the `len(caaSet.Issuewild) > 0` we are not allow to use `issue` records to make any decisions.

Reply:
@rolandshoemaker Can you cite chapter and verse for that? Section 5.3 says:
> If at least one issuewild property is specified in the relevant CAA record set, all issue properties MUST be ignored when processing a request for a domain that is a wildcard domain.

Since we're past the `if wildcard && len(caaSet.IssueWild) > 0` clause we know that this isn't a wildcard domain request and there are no issuewild properties specified in the record set. I think its reasonable to then process the issue properties, no?